pkg/api/api.go

```diff
@@ -373,6 +373,9 @@ func (a *API) registerQueryAPI(handler http.Handler) {
 func (a *API) RegisterQueryFrontend(f *frontend.Frontend) {
 	frontend.RegisterFrontendServer(a.server.GRPC, f)
 	a.registerQueryAPI(f.Handler())
+
+	// Readiness
+	a.RegisterRoute("/query-frontend/ready", http.HandlerFunc(f.ReadinessHandler), false, "GET")
```

Comment:
Should we keep this or remove it? I'm lost now ;)

Reply:
Yes, this will be removed.  Apologies!  I'm getting back to this this week.  I'm going to make this a Draft PR until I feel like it's ready again.

Reply:
No need to apologies! 🤝